test/WebJobs.Script.Tests/TestScripts/CSharp/ApiHubTableEntity/function.json

```diff
@@ -0,0 +1,17 @@
+{
+  "bindings": [
+    {
+      "type": "manualTrigger",
+      "name": "text",
+      "direction": "in"
+    },
+    {
+      "type": "apiHubTable",
+      "name": "entity",
+      "direction": "out",
+      "connection": "AzureWebJobsSql",
+      "tableName": "SampleTable",
+      "entityId": "3"
```

Comment:
Recommend changing this to "{Id}" and passing the ID in via the trigger input (e.g. "{ Id: 123 }")